interactions/api/models/guild.py

```diff
@@ -1677,6 +1677,26 @@ async def get_all_channels(self) -> List[Channel]:
         self.channels = [Channel(**channel, _client=self._client) for channel in res]
         return self.channels
 
+    async def get_all_active_threads(self) -> List[Thread]:
+        """
+        Gets all active threads of the guild.
+
+        :return: The threads of the guild.
+        :rtype: List[Thread]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+        res = await self._client.list_active_threads(int(self.id))
+        threads = [Thread(**thread, _client=self._client) for thread in res["threads"]]
+        members = [ThreadMember(**member, _client=self._client) for member in res["members"]]
+        for member in members:
+            for thread in threads:
+                if int(thread.id) == int(member.id):
```

Comment:
wait thread IDs are member IDs?

Reply:
> wait thread IDs are member IDs?

no, it's just that a `ThreadMember` has two IDs
- id: ID of the thread
- user_id: ID of the user

Reply:
Ah alr